nodes/modifier_make/solidify_mk2.py

```diff
@@ -0,0 +1,169 @@
+# ##### BEGIN GPL LICENSE BLOCK #####
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software Foundation,
+#  Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+#
+# ##### END GPL LICENSE BLOCK #####
+from mathutils import Vector
+import bpy
+from bmesh.types import BMVert, BMEdge, BMFace
+from bpy.props import FloatProperty, BoolProperty
+import bmesh
+from itertools import cycle
+from sverchok.node_tree import SverchCustomTreeNode
+from sverchok.data_structure import updateNode, zip_long_repeat
+from sverchok.utils.sv_bmesh_utils import bmesh_from_pydata, get_partial_result_pydata
+# by Linus Yng
+def create_edges(bm, v_len):
+    edges =[]
+    for edge in bm.edges[:]:
+        edges.append([v.index for v in edge.verts[:]])
+
+    edges_new = [[e[0]+v_len, e[1]+v_len] for e in edges]
+    side_edges = [[v.index, v.index+v_len] for v in bm.verts if v.is_boundary]
+    return edges + edges_new +side_edges
+
+def solidify(vertices, faces, thickness, offset=[-1], even=True):
```

Comment:
Use of mutable objects as default values of method parameters can lead to unexpected behaviour in python.